### PR TITLE
[Edit Iteration Back Button] Iter. 4.1 added tests

### PR DESCRIPTION
**Pull request by kyriakipapadimitriou**
The iterations' 'Edit' page should have a 'Back' button that will lead the user back to the Engagement's page. 
Iter. 4.1: Test have been added.

### CODE REVIEW

**Review by kimhoang73 — APPROVED** [review]
looks good!

File: features/iter_edit.feature [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  And I'm logged in on the orgs page
+  And I follow "Apps"
+  And I follow "app1"
+ 
+Scenario: There is a Back Button in the Edit Iteration page and it works
+  #Story ID: 153043640
+  Given I follow "2017-03-25"
+  And I press "Edit" for "2017-04-14"
+  Then I should see "Edit Customer Feedback"
+  And I should "Back"
```
Comment (csungwon) [inline features/iter_edit.feature]: you mean `And I should see "Back"`?

**Review by csungwon — APPROVED** [review]
I like the tests! Thanks for finding out that we did not have a back button. It would have been a serious UX issue.

File: features/iter_edit.feature (hunk 1 above)
Comment (kyriakipapadimitriou) [reply features/iter_edit.feature]: Thanks for catching that typo! I am pushing now the updated version

**Pull request merged**